24th-App-Team-1-iOS/Feature/MessageFeature/Sources/Presentation/Cell/StudentSearchTableViewCell.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  StudentSearchTableViewCell.swift
+//  MessageFeature
+//
+//  Created by 최지철 on 12/26/24.
+//
+
+import UIKit
+import DesignSystem
+
+import SnapKit
+
+public final class StudentSearchTableViewCell: UITableViewCell {
+        
+    //MARK: - Properties
+    
+    private let studentCellView = UIView()
+    private let studentImageView = UIImageView()
+    private let titleLabel = WSLabel(wsFont: .Body02, text: "학생이름")
+    private let subTitleLabel = WSLabel(wsFont: .Body06, text: "인적정보")
+    private let checkButton = UIButton().then {
```

Comment:
```suggestion
    checkButton.do {
        $0.setImage(DesignSystemAsset.Images.check.image, for: .normal)
    }
```
- checkButton에 attribute설정할때 기존 View 설정과 동일하게 `setupAttributes` 내부에서 설정하면 좋을 것 같아용!!